src/test/java/net/sf/jabref/model/database/BibDatabaseTest.java

```diff
@@ -0,0 +1,125 @@
+package net.sf.jabref.model.database;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.ParserResult;
+import net.sf.jabref.importer.fileformat.BibtexParser;
+import net.sf.jabref.model.entry.BibEntry;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Collections;
+
+import static org.junit.Assert.*;
+
+
+public class BibDatabaseTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Before
+    public void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance(); // set preferences for this test
+    }
+
+    @After
+    public void tearDown() {
```

Comment:
Yeah, but as I just noticed that doesn't matter.
`JabRefPreferences` is a Singleton; every test gets the same instance, every change in the prefs are carried to the next test.